src/entities/inputValidator/boosted/inputValidatorBoosted.ts

```diff
@@ -56,4 +64,15 @@ export class InputValidatorBoosted extends InputValidatorBase {
             }
         }
     }
+    validateCreatePool(
+        input:
+            | CreatePoolV2WeightedInput
+            | CreatePoolV2ComposableStableInput
```

Comment:
We don't have Boosted pools on V2, so you should be able to remove these 2 options